Polygon_mesh_processing/examples/Polygon_mesh_processing/cc_compatible_orientations.cpp

```diff
@@ -0,0 +1,70 @@
+#include <CGAL/Exact_predicates_inexact_constructions_kernel.h>
+#include <CGAL/Surface_mesh.h>
+
+#include <CGAL/Polygon_mesh_processing/orientation.h>
+#include <CGAL/Polygon_mesh_processing/polygon_soup_to_polygon_mesh.h>
+#include <CGAL/Polygon_mesh_processing/stitch_borders.h>
+#include <CGAL/boost/graph/IO/polygon_mesh_io.h>
+
+#include <iostream>
+#include <iterator>
+#include <map>
+#include <string>
+
+typedef CGAL::Exact_predicates_inexact_constructions_kernel Kernel;
+typedef Kernel::Point_3                                     Point;
+
+typedef CGAL::Surface_mesh<Point>                           Mesh;
+
+namespace PMP = CGAL::Polygon_mesh_processing;
+
+
+
+int main()
+{
+// create a mesh with many connected connected components that should
+// be reoriented to define a valid closed mesh
+  const std::string filename = CGAL::data_file_path("meshes/elephant.off");
+
+  Mesh mesh;
+  CGAL::IO::read_polygon_mesh(filename, mesh);
+
+  // turn the mesh into a triangle soup, duplicating all the vertices and shuffling orientations
+  std::vector<Point> points;
```

Comment:
The indentation of comments is not always align with the indentation of the code. Is that on purpose?

Reply:
Kind of, the one without leading space are the big steps, the other are intermediate comments. I don't really know how to do better, putting spaces after `//`

Reply:
If that is one purpose, then that is fine for me. I close this discussion.